compiler+runtime/include/cpp/jank/read/lex.hpp

```diff
@@ -142,6 +153,7 @@ namespace jank::read::lex
     size_t pos{};
     /* Whether or not the previous token requires a space after it. */
     native_bool require_space{};
+    native_bool found_slash_after_number{}; // True if seeing a '/' following a number.
```

Comment:
Please put comments above the line and use `/* */` style comments, like the comment on line 154.